modules/serverless-tech-preview-features.adoc

```diff
@@ -13,40 +13,50 @@ The following table provides information about which {ServerlessProductName} fea
 .Generally Available and Technology Preview features tracker
 [cols="2,1,1,1",options="header"]
 |====
-|Feature |1.31|1.32|1.33
+|Feature |1.32|1.33|1.34
 
-|OpenShift Serverless Logic
+|Eventing TLS
+|-
+|-
 |TP
+
+|Encryption Serving
```

Comment:
I suggest that we call this "Serving TLS" to align with "Eventing TLS". It's basically the same feature. Or we call both "Eventing Transport Encryption" and "Serving Transport Encryption".

Let's also remove the row "Serving TLS for internal traffic" from the "Deprecated and removed features tracker" table as it's confusing. We now have a new implementation of this TLS stuff as indicated above.

Reply:
Fixed and removed